ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -58,6 +60,20 @@ object Cli {
         .action { case (size, config) =>
           config.copy(concurrency = config.concurrency.copy(maxPoolSize = size))
         },
+      opt[String]("pem")
```

Comment:
you might want to use https://github.com/digital-asset/daml/blob/main/ledger-service/cli-opts/src/main/scala/ledger/api/tls/TlsConfigurationCli.scala instead of reimplementing this once more.

Reply:
Thanks - I wasn't aware of that util.
However, the `TlsConfigurationCli` uses object-oriented scopt DSL and the `ledger-api-bench-tool` uses functional DSL (reference: https://github.com/scopt/scopt). I've implemented functional version in the `TlsConfigurationCli` to be reused in the future.

Reply:
I would prefer if we could not have two implementations of the same helper that we need to keep in-sync. Throughout out codebase we are using the object-oriented scopt DSL. I understand it could be not as pleasing but I'd prefer if we could stick to one DSL to avoid having parts of our codebase using one approach and parts the other. If we want to revisit this choice and use the functional DSL we can talk about it, but it should be a codebase-wise effort.

My recommendation is to have a single implementation of this, using the object-oriented scopt DSL, possibly placed in a common directory (perhaps `/libs-scala`).

Reply:
fwiw afaik the main reason we are using the OO DSL is that until fairly recently (beginning of the Scala 2.13) work we were on a version of scopt that only had that. I’m very open to switching.

Reply:
I've changed the `ledger-api-bench-tool` to use OO-style parser.
Thanks for comments!